index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/main.css">
+    <link rel="stylesheet" href="./styles/nav.css">
+    <link rel="stylesheet" href="./styles/home/intro.css">
+    <link rel="stylesheet" href="./styles/home/program.css">
+    <link rel="stylesheet" href="./styles/home/instructor.css">
+    <link rel="stylesheet" href="./styles/home/partner.css">
+
+    <title>Node.js Camp</title>
+</head>
+
+<body>
+    <header>
+        <nav class="menu-secondary">
+            <ul class="seconday-list">
+                <li><a href="https://www.facebook.com/profile.php?id=100005045626812" target="_blank" rel="noopener"><img
+                            src="./images/social icons/facebook.svg" alt="facebook"></a></li>
+                <li><a href="https://twitter.com/ahmedeid2684" target="_blank" rel="noopener"><img
+                            src="./images/social icons/twitter.svg" alt="twitter"></a></li>
+                <li>English</li>
+                <li>My page</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <nav class="menu-main">
+            <img src="./images/logos/icon-only.png">
+            <ul class="menu-list">
+                <li><span class="menu-close">&times;</span></li>
+                <li class="menu-item"><a href="#">Home</a></li>
+                <li class="menu-item"><a href="./about.html">About</a></li>
+                <li class="menu-item"><a href="#program">Program</a></li>
+                <li class="menu-item"><a href="#join-btn">Join</a></li>
+                <li class="menu-item"><a href="#instructor">Instructors</a></li>
+                <li class="menu-item"><a href="#" id="menu-btn">Enroll</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section class="intro">
+            <img src="./images/mobile_hamburger.svg" alt="mobile_menu" class="mobile_menu">
```

Comment:
- [ ] Please 🙏🏽 use the `alt` attribute for the image for the best practices of HTML & CSS.

Reply:
you requested the addition of an alt attribute to certain img tag, but it appears that alt attributes already exist.

![WhatsApp Image 2023-05-24 at 6 50 00 PM](https://github.com/ahmedeid6842/Capstone/assets/57197702/d67a58f4-288f-402e-8025-1e3e59afb208)